vignettes/diving_deeper.Rmd

```diff
@@ -178,13 +182,13 @@ hematology |>
   rmarkdown::paged_table()
 ```
 
-Note that both `physical_exam` and `hematology` are **nonrepeating** instruments. REDCapTidieR **only** allows nonrepeating instruments to occur in repeating events. It will throw an error if it detects a repeating instrument associated with a repeating event.
+Note that REDCapTidieR allows for an instrument to be associated with both repeating and nonrepeating events at the same time. In this case, the the `redcap_event_instance` column will be `NA` in rows that correspond with data captured as part of a nonrepeating event.
```

Comment:
I think we still need to clarify that this is only allowed for nonrepeating instruments, this sounds like it's saying any instrument can be associated with both event structures.

Reply:
Hm, I think we're making this point at least 3 times in other places. Adding it here I think will make things less clear.

Reply:
I see no harm in a 4th for consistency, up to you though 👍